modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/HorizontalCardTag.java

```diff
@@ -73,15 +73,29 @@ public String getIcon() {
 
 	public String getTitle() {
 		String title = _title;
-
 		HorizontalCard horizontalCard = getHorizontalCard();
 
 		if ((_title == null) && (horizontalCard != null)) {
 			title = horizontalCard.getTitle();
 		}
 
-		return LanguageUtil.get(
-			TagResourceBundleUtil.getResourceBundle(pageContext), title);
+		if (isLocalized()) {
```

Comment:
I believe we need to check for `null` here as well. Something like
```suggestion
		Boolean localized = isLocalized();
		
		if ((localized != null) && localized) {
```
There might be a shorthand in Java for above, I'm not sure :sweat_smile:

Reply:
@markocikos
I think my current code meets this demand(check null solution), so we don't need to change more.
Currently isLocalized() return three value: true, false and null. We are facing with null value so I think `if (isLocalized())` is enough here. If I am wrong, please let me know.

Reply:
sorry, I forgot push my code, I did it. 
I have the same idea with you but I think the code below is enough:
```
if (isLocalized() != null) {
  inkTag.setLocalized(isLocalized());
}
```
I explained my idea above

Reply:
The part in setting `LinkTag` looks good. This is a different place in code, setting of title. You can get `if(null)` on this line. Isn't this is runtime error?

Reply:
`if(null)`, the value of _locallized will be true as default value set before.

Reply:
This line of code is in `HorizontalCardTag`, the default is `null`. If attribute was not set on tag or in modal object, `_localized` and `isLocalized()` will be `null`.

Reply:
Please note I made two comments, for two places in code. I believe you are talking about one and I'm describing the other.

Reply:
sorry my bad, I updated source.